src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 16px;
+  padding-top: 32px;
+  justify-content: center;
+  margin: 0 auto;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    font-size: 10px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-stars {
+      display: flex;
+      align-items: center;
+      gap: 4px;
+    }
+
+    &-counter {
+      line-height: 14px;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-count {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $buy-color;
+    width: 166px;
+    height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    border-radius: 5px;
+    margin: 0 auto;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    &-link {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      color: #fff;
+      width: 100%;
+      height: 100%;
+      text-decoration: none;
+
+      &:hover {
+        color: $buy-color;
+      }
+    }
+
+    &:hover {
+      color: $buy-color;
+      background-color: #fff;
+      border: 1px solid $buy-color;
```

Comment:
The variable `$buy-color` is used here as well. Make sure it is defined in a separate SCSS file for variables.